### DISCUSSION TITLE
Run error on  Enterpise x64

### DISCUSSION THREAD

**Discussion by theycallmevirgo** [winfsp/winfsp#446] [Q&A]
On win11 Enterprise x64 using vs2022,  ntpfs solution builds without error as debug but when attempting to run, returns error 

`The service ptfs has failed to start (Status=c0000001).`

Thanks so much for any help

Joe

ETA: Release returns identical error

**Comment by billziss-gh** [comment]
The `c0000001` status code is `STATUS_UNSUCCESSFUL`. Ntptfs usually returns this error when it also reports a command line usage error.

If this is not your case, perhaps you can step through the program with a debugger and see which API fails with this error?
